### PR TITLE
Add new project

### PR DESCRIPTION
**Pull request by neilbulloch**
created form page to add new project, created css for form page, added upload file functionality and upload folder

### CODE REVIEW

File: upload.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+//    echo "Sorry, your file is too large.";
+//    $uploadOk = 0;
+
+// Allow certain file formats
+//if($imageFileType != "jpg" && $imageFileType != "png" && $imageFileType != "jpeg"
+//    && $imageFileType != "gif" ) {
+//    echo "Sorry, only JPG, JPEG, PNG & GIF files are allowed.";
+//    $uploadOk = 0;
+//}
+// Check if $uploadOk is set to 0 by an error
```
Comment (Liliya-V) [inline upload.php]: Do you need all these commented lines?

File: upload.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+//if ($_FILES["fileToUpload"]["size"] > 500000) {
+//    echo "Sorry, your file is too large.";
+//    $uploadOk = 0;
+
+// Allow certain file formats
+//if($imageFileType != "jpg" && $imageFileType != "png" && $imageFileType != "jpeg"
+//    && $imageFileType != "gif" ) {
+//    echo "Sorry, only JPG, JPEG, PNG & GIF files are allowed.";
+//    $uploadOk = 0;
+//}
```
Comment (Liliya-V) [inline upload.php]: Do you need all these commented lines?

File: upload.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+// Check if $uploadOk is set to 0 by an error
+if ($uploadOk == 0) {
+    echo "Sorry, your file was not uploaded.";
```
Comment (Liliya-V) [inline upload.php]: Sorry file exists and Sorry file was not uploaded appear same time ( to fix in a next task?)

File: newProject.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
+
+
+
```
Comment (Liliya-V) [inline newProject.php]: delete extra lines

File: CSS/newProject.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    border-radius: 4px;
+    font-size: 1rem;
+    transition: width 0.7s ease-in-out;
+    padding: 5px;
+}
+
+input[type=text]:focus {
+    width: 70%;
+}
+.addNewForm input[type=file] {
```
Comment (Liliya-V) [inline CSS/newProject.css:52]: add a line to follow css standard

File: CSS/newProject.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    background-color: #F75A5C;
+    border: none;
+    color: white;
+    padding: 16px 16px;
+    text-decoration: none;
+    margin: 4px 2px;
+    cursor: pointer;
+    font-size: 1rem;
+    border-radius: 20px;
+}
```
Comment (Liliya-V) [inline CSS/newProject.css:62]: add a line to follow css standard

File: CSS/newProject.css [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+input[type=text]:focus {
+    width: 70%;
+}
+
+.addNewForm input[type=file] {
+    background-color: #F75A5C;
+    border: none;
+    color: white;
```
Comment (mporam) [inline CSS/newProject.css]: use hex everywhere, no color names

File: newProject.php [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+<body>
+<div class="logo">
+    <img src="Images/logo_transparent.png" alt="logo">
+</div>
+<h1>Add new entry</h1>
+
+<div class="addNewForm">
+
+    <form method = "POST" action="upload.php" enctype="multipart/form-data">
+        <h3>Project Title: </h3>
```
Comment (mporam) [inline newProject.php]: use `<label>` tags instead of h3

File: upload.php [hunk 9]
```diff
@@ -0,0 +1,54 @@
+<?php
+/**
```
Comment (mporam) [inline upload.php]: remove unnecessary comment

File: upload.php [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 2019-03-18
+ * Time: 20:35
+ */
+$db = new PDO("mysql:host=192.168.20.20;dbname=Portfolio", 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+//POST data from add new project form
+$title = $_POST['title'];
```
Comment (mporam) [inline upload.php]: no check to see if title or site exist?

File: upload.php [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+$query->bindParam(':site', $site, PDO::PARAM_STR );
+
+//php to upload image to server
+$file_path = "Images/";
+$file_path = $file_path . basename($_FILES["fileToUpload"]["name"]);
+$uploadOk = 1;
+$imageFileType = strtolower(pathinfo($file_path,PATHINFO_EXTENSION));
+
+// Check if image file is a actual image or fake image
+if(isset($_POST["submit"])) {
```
Comment (mporam) [inline upload.php]: PSR2 says:
`if (`

**Review by mporam — CHANGES_REQUESTED** [review]

File: upload.php [hunk 12]
```diff
@@ -0,0 +1,57 @@
+<?php
+$db = new PDO("mysql:host=192.168.20.20;dbname=Portfolio", 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+//POST data from add new project form
+$title = $_POST['title'];
+$site = $_POST['site'];
+
+if (empty($_POST['title'])) {
```
Comment (mporam) [inline upload.php]: this doesnt work

**Review by mporam — CHANGES_REQUESTED** [review]

**Pull request merged**